scripts/BONSpecialGenerator.swift

```diff
@@ -147,6 +154,8 @@ if sortedSpecialCharacters != specialCharacters {
 var headerEnumString = "typedef NS_ENUM(unichar, BONCharacter) {\n"
 var headerCodeString = ""
 var implementationCodeString = ""
+var humanReadableDictionaryHeaderDeclaration = "+ (NSDictionary *)humanReadableStringDictionary;"
```

Comment:
#pp declare these closer to where they're used
#req `let`, not `var`
#req for both header and implementation: this should use `BONGeneric` to expose a strongly typed dictionary.